mono/tests/tailcall/Makefile

```diff
@@ -0,0 +1,54 @@
+# Tests are written in C# to be amenable to tail calls.
```

Comment:
Do you really need custom Makefile?
Most of the dependent tools are wrong check for example https://github.com/mono/mono/blob/master/mono/tests/Makefile.am#L84 and it'll also make it harder to include in existing test suites